src/app/characters/characters.service.ts

```diff
@@ -21,15 +25,18 @@ export class CharacterService {
 
   constructor(private http: HttpClient, private contextService: ContextService) {}
 
-  getCharacters(term: string): Observable<Character[]> {
-    const options = new HttpParams().set('nameStartsWith', term);
+  getCharacters(term: string, offset = 0, limit = 20): Observable<MarvelResponse['data']> {
```

Comment:
Since we use the Marvel API with default values for `offset` and `limit`, can we get rid of them?

Reply:
The only reason I left them here is for future use. They also serve so that one can understand the readonly variable (`readonly CHARACTERS_LIMIT = 20;`) at the `CharacterListComponent` component and not be considered as a "magic" number. But if you think it makes the code more complicated, we can remove it. What do you think;

Reply:
>The only reason I left them here is for future use

One of the best practices that I like to follow is to do something as soon as there is an actual need for that. Because in the future we may follow a different approach and the code that we have written will result to be obsolete 🙂 So, at least `offset` is not needed in this case.

For the `CHARACTERS_LIMIT`, what I would do, would be to add it in the `environment` object and reference it directly from there in the component and the service. Environments are a good fit for this case because in the default environment (dev), it would make sense to have a larger page size for development and testing purposes.

Reply:
Great idea to move it at `environment`. Done.